src/xenia/ui/window.cc

```diff
@@ -180,6 +182,33 @@ void Window::OnPaint(UIEvent* e) {
         (static_cast<double>(now_ns - fps_update_time_ns_) / 10000000.0));
     fps_update_time_ns_ = now_ns;
     fps_frame_count_ = 0;
+
+#if XE_OPTION_PROFILING
+    // This means FPS counter will not work with profiling disabled (e.g. on
+    // Linux)
+    float fToMs =
+        MicroProfileTickToMsMultiplier(MicroProfileTicksPerSecondCpu());
+    uint64_t nFlipTicks = 0;
+    {
+      std::lock_guard<std::recursive_mutex> lock(MicroProfileGetMutex());
+      MicroProfile& S = *MicroProfileGet();
+      nFlipTicks = S.nFlipTicks;
+    }
+    float fMs = fToMs * nFlipTicks;
+
+    if (fMs != 0.0f) {
+      game_fps_ = static_cast<uint32_t>(1000.0f / fMs);
+    }
+#endif
+
+    title_fps_text_ = base_title_;
+    title_fps_text_ += L" | ";
```

Comment:
Maybe an option for the user to toggle showing the FPS number in the titlebar.
Since applications like OBS won't be able to hook due to the title kept changing. 2/2
```suggestion
    if (cvars::fps_titlebar) {
      title_fps_text_ += L" | ";
      title_fps_text_ += std::to_wstring(game_fps_);
      title_fps_text_ += L" FPS";
    }